### PR TITLE
WIP Print through app UI

### PR DESCRIPTION
**Pull request by SuaYoo**

### CODE REVIEW

File: src/npld-player.ts [hunk 1]
```diff
@@ -210,7 +214,15 @@ export class NPLDPlayer extends LitElement {
...
 
-  private printPage() {
-    console.log('TODO printPage');
+  // https://www.electronjs.org/docs/latest/api/webview-tag#webviewprintoptions
+  private async printPage() {
+    try {
+      // const res: Uint8Array = await this.webview.printToPDF({});
+      // console.log(res);
+
+      await this.webview.print();
```
Comment (SuaYoo) [inline src/npld-player.ts:223]: @ikreymer do you have experience printing the webview? This resolves without actually bringing up the print dialog for me. I don't have any printers connected, but expected to at least see a system print dialog. Don't see any related issues, I wonder if it's related to my system or I'm missing something from the documentation--any ideas?

**Comment by GilHoggarth** [comment]
@ikreymer @SuaYoo What is the status of this PR, is it ready to be merged into the main branch?
I note there is a Conflict regarding `src/npld-player.ts` - can this be resolved please.
Gil

**Comment by tw4l** [comment]
Hi @GilHoggarth, based on the unresolved comment above the PR title, I wouldn't expect this feature to be fully ready. Also worth noting that pywb 2.8 adds a print button into the pywb header, which might also work in this context? Could likely use more investigation, apologies for the slow responses here.

**Comment by GilHoggarth** [comment]
Thanks @tw4l, and of course you're right - tried to force a build to learn the status, and I've learnt it's not ready. Well, at least I now know.

**Pull request merged**